R/tidyverse.R

```diff
@@ -222,12 +222,14 @@ use_tidy_style <- function(strict = TRUE) {
   check_installed("styler")
   check_uncommitted_changes()
   if (is_package()) {
-    styled <- styler::style_pkg(proj_get(),
-                                style = styler::tidyverse_style, strict = strict
+    styled <- styler::style_pkg(
+      proj_get(),
+      style = styler::tidyverse_style, strict = strict
```

Comment:
This still isn't quite right because `strict `should be on a new line

Reply:
Fixed in #297.